src/services/storage/hooks/offers.hooks.ts

```diff
@@ -101,7 +128,8 @@ export default {
             attributes: {
               include: [
                 [getBillingPriceAvgQuery(sequelize, 'usd'), 'avgBillingPrice'],
-                [getStakesAggregateQuery(sequelize, 'usd'), 'totalStakedUSD']
+                [getStakesAggregateQuery(sequelize, 'usd'), 'totalStakedUSD'],
+                [getAvailableCapacityQuery(), 'availableCapacity']
```

Comment:
Shouldn't be this added only when `availableCapacity`?

Reply:
Hmm I don't think so because this is [replacing the virtual column `availableCapacity`](https://github.com/rsksmart/rif-marketplace-cache/pull/414/files#diff-494e5b8a4ef33c59f54a87ed47867e8cf912680e6a00339a3af0cf2514ed327dL50) that was always provided when fetching offers. So we should keep providing this value always, right?

Reply:
Sounds good!